src/app/components/landingPage/Partners.tsx

```diff
@@ -0,0 +1,24 @@
+import Image from 'next/image';
+
+import {partners} from '../../../data/config/landingPageConfig';
+
+import {Card} from './Card';
+import {ListBloc} from './ListBloc';
+
+import styles from '../../../../styles/app/layout/landing/parterAndSponsors.module.css';
+
+export const Partners = () => {
+	return (
+		<ListBloc sectionTitle="They worked with us">
+			{partners.map(({name, logo, url}) => (
+				<a key={name} href={url} target="_blank">
```

Comment:
**suggestion** Why won't we use a next link here ?

Reply:
Forgot tu use Link here thanks